src/chain_sync/tipset_syncer.rs

```diff
@@ -1213,11 +1206,11 @@ async fn validate_tipset<DB: Blockstore + Send + Sync + 'static, C: Consensus>(
 /// * Checking that the messages in the block correspond to the agreed upon
 ///   total ordering
 /// * That the block is a deterministic derivative of the underlying consensus
-async fn validate_block<DB: Blockstore + Sync + Send + 'static, C: Consensus>(
-    consensus: Arc<C>,
+async fn validate_block<DB: Blockstore + Sync + Send + 'static>(
+    consensus: Arc<FilecoinConsensus>,
```

Comment:
The `consensus` argument is not required here. You can create a new `FilecoinConsensus` structure from the beacon in the state manager. Removing the `consensus` argument will have a ripple effect on the rest of the code.